uk_bin_collection/uk_bin_collection/councils/CotswoldDistrictCouncil.py

```
@@ -4,240 +4,152 @@
+                            except (ValueError, AttributeError):
+                                continue
                             data["bins"].append({
                                 "type": container_type,
-                                "collectionDate": collection_date
+                                "collectionDate": parsed_date,
+                            })
+                    except (ValueError, AttributeError):
+                        continue
+            else:
+                body_text = driver.find_element(By.TAG_NAME, "body").text
+                lines = [l.strip() for l in body_text.split("\n") if l.strip()]
```

Ruff (0.15.15)

[error] Ambiguous variable name: `l`

(E741)